practice-two/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "practice-two",
     "description": "Build the UI of a page with React",
+    "type": "module",
```

Comment:
why we need it? does it work with precommit or sth else

Reply:
Yes, if the line is missing, the npm start command to run the server will not work, i think precommit it not working

Reply:
`(node:10432) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
[Failed to load PostCSS config: Failed to load PostCSS config (searchPath: D:/Git-repo/Thehienduong-git/Duong-The-Hien-ReactJS/practice-two): [SyntaxError] Unexpected token 'export'
D:\Git-repo\Thehienduong-git\Duong-The-Hien-ReactJS\practice-two\postcss.config.js:1
export default {
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1088:15)
    at Module._compile (node:internal/modules/cjs/loader:1123:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)]

Node.js v18.12.1`